.scripts/Generate-Api.ps1

```diff
@@ -0,0 +1,49 @@
+$repoRoot = Join-Path $PSScriptRoot .. -Resolve
+$dotnetFolder = Join-Path $repoRoot .dotnet\src
+$apiFolder = Join-Path $repoRoot .dotnet\api
+
+$platformTarget = "netstandard2.0"
+
+$assemblyPath = Join-Path $dotnetFolder bin\Debug $platformTarget OpenAI.dll
```

Comment:
Can we add a `dotnet build` step before this so that we get the latest dll, which will generate the latest API?

Reply:
+1. This also helps guarantee that you're generating the API from a DLL that is up-to-date.